packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/matches..st

```diff
@@ -0,0 +1,10 @@
+testing
+matches: aMarkdownDelimiterEmphasis
```

Comment:
Discuss:
`matches` is very generic. Maybe we could use `isMatchingOpenerFor: aClosingDelimiterEmphasis`?